ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/router-bgp-base.cfg

```diff
@@ -13,11 +13,14 @@ interface Management1
    ip address 10.73.255.122/24
 !
 router bgp 65101
-   no bgp default ipv4-unicast
```

Comment:
Why is this line getting removed?

Reply:
Hi Claus,

I don't think it's removed, just the order changes slightly:

```
router bgp 65101
   router-id 192.168.255.3
   distance bgp 20 200 200
   maximum-paths 32 ecmp 32
   update wait-for-convergence
   update wait-install
   **no bgp default ipv4-unicast**
   graceful-restart restart-time 300
   graceful-restart
```

due to how I arranged it.

In the current code, most of these commands are kept under a single key:

```
  bgp_defaults:
    - no bgp default ipv4-unicast
    - distance bgp 20 200 200
    - graceful-restart restart-time 300
    - graceful-restart
    - maximum-paths 2 ecmp 2
 ```

and I've removed a couple of them and defined them explicitly instead, so they appear on top now..

Reply:
can we keep it like that, or do I need to adjust the order?

Reply:
My bad, I missed it.